dna.json

```diff
@@ -4,5 +4,11 @@
      ],
     "casks": [
     ]
+  },
+  "chocolatey": {
+    "packages": [
+      "conemu",
+      "git"
```

Comment:
While it's delightful to have these things installed, the package arrays are empty so we don't have a preconceived opinion that may not align with the end user. There will be another thing users can do to feed this in an easily customizable way.